projects/transformers/run_utils.py

```diff
@@ -708,6 +676,22 @@ def init_model(model_args, config, tokenizer, finetuning=False):
     return model
 
 
+def toggle_drop_last_factory(method):
```

Comment:
I believe this method is described as a wrapper than a factory. Would you agree?

Reply:
I like this solution btw. Seems like a good way to modify the function without having to create a new subclass to Trainer.